frontend/packages/console-shared/src/components/dynamic-form/templates.tsx

```diff
@@ -1,119 +1,45 @@
 import * as _ from 'lodash';
-import * as classnames from 'classnames';
 import * as React from 'react';
 import {
   ArrayFieldTemplateProps,
   FieldTemplateProps,
   ObjectFieldTemplateProps,
-  FieldProps,
 } from 'react-jsonschema-form';
-import {
-  AccordionItem,
-  AccordionToggle,
-  AccordionContent,
-  Button,
-  Alert,
-} from '@patternfly/react-core';
+import { Button, Alert, FormHelperText } from '@patternfly/react-core';
 import { MinusCircleIcon, PlusCircleIcon } from '@patternfly/react-icons';
 import { JSON_SCHEMA_GROUP_TYPES } from './const';
 import { getUiOptions, getSchemaType } from 'react-jsonschema-form/lib/utils';
 import { ExpandCollapse } from '@console/internal/components/utils';
-
-export const FieldLabel: React.FC<FieldLabelProps> = ({ htmlFor, label, required }) => (
-  <label className={classnames('form-label', { 'co-required': required })} htmlFor={htmlFor}>
-    {label}
-  </label>
-);
-
-export const FormField: React.FC<FormFieldProps> = ({
-  children,
-  displayTitle = true,
-  id,
-  required,
-  title,
-}) => {
-  return (
-    <div id={`${id}_field`} className="form-group">
-      {displayTitle && <FieldLabel label={title || 'Value'} required={required} htmlFor={id} />}
-      {children}
-    </div>
-  );
```

Comment:
This is just from moving `FieldSet` and `FormField` components to the `fields.tsx` file.